doc/theoretical_description_multilabel_classification.rst

```diff
@@ -7,16 +7,16 @@ Theoretical Description
 =======================
 
 
-Two methods for multi-label uncertainty-quantification have been implemented in MAPIE so far :
-Risk-Controlling Prediction Sets (RCPS) [1] and Conformal Risk Control (CRC) [2].
+Three methods for multi-label uncertainty-quantification have been implemented in MAPIE so far :
+Risk-Controlling Prediction Sets (RCPS) [1], Conformal Risk Control (CRC) [2] and Learn Then Test (LTT) [3].
 The difference between these methods is the way the conformity scores are computed. 
```

Comment:
QUESTION: is it the case for LTT?

Reply:
Yes Learn Then Test is a method for  uncertainty quantification as RCPS or CRC

Reply:
Is it a method for control risk ? Or for conformity score computation?